app/src/main/java/com/babylon/wallet/android/domain/usecases/GetFreeXrdUseCase.kt

```diff
@@ -44,7 +44,7 @@ class GetFreeXrdUseCase @Inject constructor(
             val manifest = BabylonManifestBuilder()
                 .lockFee()
                 .freeXrd()
-                .accountTryDepositBatchOrAbort(
+                .accountTryDepositEntireWorktopOrAbort(
```

Comment:
is the `accountTryDepositEntireWorktopOrAbort` a renaming of the  `accountTryDepositBatchOrAbort` or just another manifest instruction (I mean inside the library)? And what's the difference? Or just point me to some documentation if there is any

Reply:
Sorry didn't find any documentation. Derived it myself after trying to make the faucet work. The `accountTryDepositBatchOrAbort`, after the update, had a new parameter of a list of buckets. So that means that for the try deposit to work, we needed a reference to some buckets. On the other hand in this specific manifest, what we actually care about is to deposit all worktop xrd into the user's account address. Since we are considering worktop we don't have to reference any bucket. So it was more suited.